release/ray_release/config.py

```diff
@@ -40,12 +40,13 @@
 
 
 def read_and_validate_release_test_collection(
-    config_files: List[str], schema_file: Optional[str] = None
+    config_files: List[str],
+    schema_file: Optional[str] = None,
 ) -> List[Test]:
     """Read and validate test collection from config file"""
     tests = []
     for config_file in config_files:
-        with open(config_file, "rt") as fp:
+        with open(bazel_runfile(config_file), "rt") as fp:
```

Comment:
[1]

Reply:
not sure if this will work though..

Reply:
discuss offline, bazel_runtile returns an absolute that works both in bazel runtime and python runtime environment